### PR TITLE
Windows 10 SDK is also required now.

### PR DESCRIPTION
**Pull request by sfengyuan**

### CODE REVIEW

**Comment by retep998** [comment]
The Windows SDK was always required.

**Comment by jacobsun** [comment]
> The Windows SDK was always required.

But why the guide and the Book did not mention that? It just waste  lots of [peoples](https://www.google.com/search?newwindow=1&ei=e3D3XIXoL6a4mAWqyYLIDg&q=LINK++cannot+open+input+file+%27advapi32.lib%27&oq=LINK++cannot+open+input+file+%27advapi32.lib%27&gs_l=psy-ab.3..0i7i30.152589.153222..153851...0.0..0.340.578.2-1j1......0....1..gws-wiz.EhMrUGmHYaI) time.

**Comment by alexcrichton** [comment]
It's true that Rust has alwasy required this, but @jacobsun did you perhaps have Visual Studio but without the required components installed? If so perhaps the components could be listed here to be sure to mention that they're required?

**Comment by jacobsun** [comment]
I had tried to install c++ build tools without vs,  vs with c++ build tools, and finally vs with c++ build tools and  windows 10 sdk.

File: README.md [hunk 1]
```diff
@@ -132,6 +132,8 @@ MSVC builds of Rust additionally require an installation of Visual Studio 2017
 (or later) so `rustc` can use its linker.  The simplest way is to get the
 [Visual Studio Build Tools] and check the “C++ build tools” workload.
 
+Windows 10 SDK is also required now.
```
Comment (steveklabnik) [inline README.md]: We would certainly want to phrase this in the present, that is, not "now".

File: README.md [hunk 2]
```diff
@@ -130,11 +130,10 @@ build.
 
 MSVC builds of Rust additionally require an installation of Visual Studio 2017
 (or later) so `rustc` can use its linker.  The simplest way is to get the
-[Visual Studio Build Tools] and check the “C++ build tools” workload.
+[Visual Studio], check the “C++ build tools” and “Windows 10 SDK” workload.
 
-Windows 10 SDK is also required now. 
 
-[Visual Studio Build Tools]: https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019
+[Visual Studio]: https://visualstudio.microsoft.com/zh-hans/downloads/
```
Comment (jacobsun) [inline README.md]: The old link is redirected to vs 2019, and there is not a standalone C++ build tools.

File: README.md (hunk 1 above)
Comment (jacobsun) [reply README.md]: The link is also changed. There isn't a c++ build tools.

File: README.md (hunk 2 above)
Comment (mati865) [reply README.md]: https://visualstudio.microsoft.com/downloads/ is language neutral link.

Comment (jacobsun) [reply README.md]: I shouldn't paste that url.

**Pull request merged**